BJS2-92277 Solution of the task

KuaNySH88 opened the pull request.

KuaNySH88 committed:
BJS2-92277 Solution of the task

SnurnicynKonstantin commented on src/main/java/school/faang/bjs2_92277/Main.java:
```diff
@@ -0,0 +1,23 @@
+
+public class Main {
+    private static final int TIME_FOR_SLEEP = 5000;
+    public static boolean is_game_over = false;
```
is_game_over - Это что за Python-stype именования переменных в java)))

KuaNySH88 replied:
поправил

SnurnicynKonstantin commented on src/main/java/school/faang/bjs2_92277/Main.java:
```diff
@@ -0,0 +1,23 @@
+        boolean isPointsAdded;
+        boolean isLifeMinused;
+
+        while (!is_game_over) {
```
А где тут многопоточка?)
В цикле операции друг за другом будут выполняться. Ситуации гонки не возникнет никогда. Следовательно synchronized не нужны будут.
Надо udate в различных потоках вызывать, через ExecutorService желательно.

KuaNySH88 replied:
добавил

KuaNySH88 committed:
BJS2-92277 Fix PR comments:
1 change python style on java style in naming of the variable isGameOver
2 add multithreading

SnurnicynKonstantin reviewed (approved):
Вот теперь через executorService очень корректно получилось)
